### DISCUSSION TITLE
[Script request] Komga

### DISCUSSION THREAD

**Discussion by yry-dev** [community-scripts/ProxmoxVE#270] [Request script] [answered]
### Application Name

Komga

### Website

https://komga.org

### Description

A media server for your comics, mangas, BDs, magazines and eBooks

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by yry-dev** [comment]
I have already done the work if someone wants to help me test it
```
bash -c "$(wget -qLO - https://raw.githubusercontent.com/DysfunctionalProgramming/ProxmoxVE/refs/heads/komga/ct/komga.sh)
```
Here is the source:
https://github.com/DysfunctionalProgramming/ProxmoxVE/pull/2

Sorry if this is the wrong way to do this, I didn't really see a contributor guide for this project.

**Reply by MickLesk** [reply]
Its okay, but they have some issues in the script.
You can create a PR to this main project and we check it. 

Ive checked the ct\sh, and the update is not really good :P there are some improvements needed

**Comment by MickLesk** [comment, marked as answer]
Komga LXC
Date added: 2024-11-15

```bash
bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/komga.sh)"
